integration/testrunner/public_key.json

```diff
@@ -0,0 +1,4 @@
+{
+  "encoded": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhBWj8vw5LkPRWbCr45k0cOarIcWg\nApM03mSYF911de5q1wGOL7R9N8pC7jo2xbS+i1wGsMiz+AWnhhZIQcNTKg==\n-----END PUBLIC KEY-----\n",
```

Comment:
If there is no specific reason for go to use encoded public key, it might be better just to use the decoded public key.

Reply:
There are something tricky for go to use the decoded public key, I am working on that, but I think I can fix it in another PR.

Reply:
If you wanna fix it in another PR, create a new github issue and add TODO in the code.